### ISSUE TITLE
[BUG] Seg Fault in E2E_Example.ipynb

### ISSUE THREAD

**Issue by mnicely** [rapidsai/cusignal#172]
While implementing #101 , the follow issue appeared in CI

```bash
/var/lib/jenkins/workspace/rapidsai/gpuci/cusignal/prb/cusignal-gpu-build/tmp/E2E_Example-test.py:239: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  return F.log_softmax(x)
bash: line 1: 17120 Segmentation fault      (core dumped) ipython --colors=NoColor /var/lib/jenkins/workspace/rapidsai/gpuci/cusignal/prb/cusignal-gpu-build/tmp/E2E_Example-test.py
-------------------------------------------------------------------------------
DONE: E2E_Example

real	1m32.684s
user	1m10.780s
sys	0m20.944s

EXIT CODE: 139
```

It looks like `log_softmax` has be replaced with `softmax`?
